### DISCUSSION TITLE
Forget the address of crazyflie2.1, how to reset to 0xE7E7E7E7E7 ?

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1644] [General]
Hi,

Forget the address of crazyflie2.1, how to reset to 0xE7E7E7E7E7 ?

**Comment by NageshEranki** [comment]
Hi Justin

Here are the [instructions](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#firmware-configuration) to redefine radio channel/address. You will see that you need to first connect the drone in CFclient. However, since you dont remember the radio address, you can connect via USB and proceed with the instructions.

Hope this helps!

**Comment by justinleeyang** [comment]
Hi, NageshEranki：

follow the [instructions](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#firmware-configuration), I try to write the address, but reboot the crazyflie2.1, still don't scan the device, Is there any other method or how to read the address information of  crazyflie2.1 ？

**Reply by Bananazooka** [reply]
Hi, have you connected via USB? Have you followed those [steps](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/) too?

**Reply by justinleeyang** [reply]
Hi,  I haved connected via USB, and follow the [instructions](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/#firmware-configuration)

**Reply by Bananazooka** [reply]
Hi, did you also follow those [instructions ](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/installation/install/ )to install the client and the radio drivers?
